[WIP] fix mistakes (#146)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-android/overview.md

## Main screen

> There are two bars on the main screen, reflecting the app status (Connected/Disconnected) and the selected Exclusions mode ([General/Selective](#lists-of-exclusions)). On the same screen, there are also a *Connect/Disconnect* button and a list of available servers.

Reviewer: connection status как будто корректнее тут чем app status

Reviewer: и еще! исключения больше не отображаются на главном

> Each server has its location and its ping rate, describing the response time of the server. The lower this rate, the faster the connection. The fastest servers always appear at the top of the list, which consists of more than 50 locations in dozens of countries. You can connect to the fastest server by tapping the *Connect/Disconnect* button or by picking a location.

Reviewer: запятая была лишняя, но тут в целом не очень правильное употребление слова which
лучше that и без запятой

Reviewer: еще момент: странно писать You can connect to the fastest server by tapping the *Connect/Disconnect* button or by picking a location.
чтобы подключиться, можно только Connect нажать, Disconnect не подойдет

## Exclusions

We've done everything to make it easy for you to manage your site and app exclusion lists. AdGuard VPN will only work where you choose.

### Lists of exclusions

> The exclusion lists allow you to select sites for which the VPN should be enabled — and vice versa. To reach the *Exclusions* section, tap the second icon from the left at the bottom of the screen.

Reviewer: не уверена, что vice versa тут правильно. мне кажется, оно было бы корректнее в ситуации «a влияет на b и наоборот»

плюс приложения не надо забывать

поэтому предложила бы что-то другое

The exclusion lists allow you to manage the VPN connection for specific websites and apps. To access *Exclusions*, tap the second icon from the left at the bottom of the screen.

:::

> As you can enable subdomains in service lists, we added boxes that reflect the status of each service — you can see them on the main screen of *Exclusions* to the left of each service name: **fully-enabled** status is marked with a white checkmark on green background, **fully-disabled** — with a gray box, and **partly-enabled**, which means that one or more parameters were changed — with a green square on white background. Good news: you can always return to the default view of service lists in case you’ve deleted or disabled any domains from there.

Reviewer: сомневаюсь насчет такого употребления тире. хочется тут списком сделать + местами бы поменяла
кстати гугл говорит что американский вариант все же check mark

* **Fully enabled** is indicated by a white check mark on a green background
* **Partially enabled** (enabled subdomains without the main domain) is marked with a green square on a white background
* **Fully disabled** is marked with a blank checkbox

> ### Apps Settings

Reviewer: ```suggestion
### App exclusions
```

Reviewer: но лучше объединить с предыдущим

As we mentioned above, not only websites can be easily added to the exclusions. Choose for which apps you need AdGuard VPN — and vice versa. Tap the icon next to the *Exclusions lists* icon at the bottom of the screen, to open Apps settings. By default, AdGuard VPN works for all apps, but the slider can be toggled — thus turning off AdGuard VPN — for any app on the list.

> If the *Compatibility mode* with AdGuard is enabled, you can only manage apps through the AdGuard Ad Blocker. Therefore, when you tap the button, the AdGuard app opens.

Reviewer: ```suggestion
In *Integrated mode*, you can only manage apps through AdGuard Ad Blocker.
```

![Import/Export *mobile_border](https://cdn.adguardvpn.com/content/kb/vpn/android/imp-exp.png)

> ### For apps

Reviewer: ```suggestion
#### For apps
```
чтобы на одном уровне с сайтами было

As we mentioned above, not only websites can be easily added to the exclusions. Choose for which apps you need AdGuard VPN and for which you don't. By default, AdGuard VPN works for all apps, but you can easily switch to the other mode.

In *Integrated mode*, you can only manage apps through AdGuard Ad Blocker.

> ![App exclusions *mobile_border](https://cdn.adguardvpn.com/content/kb/vpn/android/apps_settings.png)

Reviewer: скрин надо будет не забыть поправить

### DNS servers

> The purpose of the [Domain name system](https://adguard-dns.io/kb/general/dns-filtering/#what-is-dns) (DNS) is to translate websites' names into something browsers can understand, i.e. IP addresses. This job is performed by DNS servers. AdGuard VPN for Android offers a wide selection of DNS servers, each with special qualities. For example, [AdGuard DNS](https://adguard-dns.io/kb/) removes ads and protects your device from tracking while AdGuard DNS Family Protection combines the functions of AdGuard DNS with SafeSearch and adult content blocking. There is also an option to add a custom DNS server.

Reviewer: я бы убрала вступление нафиг

Reviewer: а зачем оно тут вообще...((

### Advanced settings

> In the *Advanced settings*, you can find five sections. You can *Help us to become better* by toggling the switch in the upper block. This action will allow AdGuard VPN to gather crash reports, technical and interaction data. Submission will be in the form of anonymous information.

Reviewer: точно надо the Advanced settings?

Reviewer: *Help us to become better* больше нет... а настройка вместо нее теперь в дженерал

> *Operating mode* section allows you to choose one of three options: VPN, Proxy, and Compatibility mode. In *VPN mode* all traffic is routed through AdGuard VPN automatically. When the *Proxy mode* (SOCKS5) is on, AdGuard VPN runs a local proxy server through which traffic from other apps can be routed. Choose this option only if you know what you are doing. Enabled *Compatibility mode* allows AdGuard VPN and AdGuard Ad Blocker to work together.

Reviewer: ```suggestion
*Operating mode* allows you to specify how your traffic is routed. There are three modes: VPN, SOCKS5, and Integrated mode. In the *VPN* mode, all traffic is routed through AdGuard VPN. In the *SOCKS5* mode, AdGuard VPN runs a local proxy server that can be used by other apps for traffic routing. *Integrated mode* allows AdGuard VPN and AdGuard Ad Blocker to work together.
```
